makefile.shade

```diff
@@ -60,6 +60,7 @@ var MANAGED_PROJECTS = '${FindAllProjects(
     "Microsoft.Framework.PackageManager",
     "Microsoft.Framework.Project",
     "Microsoft.Framework.Runtime",
+	"Microsoft.Framework.Runtime.Caching",
```

Comment:
Correct. When I started the refactoring, it wasn't (I was trying to see exactly where the caching was used). I'll remove this.